easy_infra.yml

```diff
@@ -5,6 +5,7 @@ _anchors:
     checkov:
       command: checkov -d . --download-external-modules True --no-guide
       customizations:
+        CHECKOV_BASELINE: --baseline
```

Comment:
This is the only thing needed to add `--baseline` support to Checkov.  Everything else is documentation and tests.